src/decoders/router_decoder_custom_worker.erl

```diff
@@ -107,13 +135,32 @@ init(Args) ->
 %%         )
 %%     of
 %%         {error, invalid_context} ->
+%%             %% Execution reaching here implies V8 vm restarted recently
 %%             {Delay, Backoff1} = backoff:fail(Backoff0),
+%%             %% Refresh their Context and JS definition:
 %%             _ = erlang:send_after(Delay, self(), ?INIT_CONTEXT),
 %%             {reply, {error, invalid_context}, State1#state{
 %%                 context = undefined,
 %%                 backoff = Backoff1
 %%             }};
-%%         {error, _} = Error ->
+%%         {error, Reason0} = Error ->
```

Comment:
Does this big ol' comment get to go away? 🙏

Reply:
Fixed in commit 731a85b